tests/test_config_loader.py

```diff
@@ -593,6 +593,38 @@ def test_sweep_config_cache(
         monkeypatch.setenv("HOME", "/another/home/dir/")
         assert sweep_cfg.home == os.getenv("HOME")
 
+    @pytest.mark.parametrize(  # type: ignore
+        "key,expected",
+        [
+            pytest.param("id123", "id123", id="id"),
+            pytest.param("123id", "123id", id="int_plus_id"),
+            pytest.param("'quoted_single'", "quoted_single", id="quoted_single"),
+            pytest.param('"quoted_double"', "quoted_double", id="quoted_double"),
+            pytest.param("'quoted_$(){}[]'", "quoted_$(){}[]", id="quoted_misc_chars"),
+            pytest.param("a/-\\+.$%*@", "a/-\\+.$%*@", id="unquoted_misc_chars"),
+            pytest.param("white space", "white space", id="whitespace"),
+            pytest.param(
+                "\\\\\\(\\)\\[\\]\\{\\}\\:\\=\\ \\\t\\,",
+                "\\()[]{}:= \t,",
+                id="unquoted_esc",
+            ),
+        ],
+    )
+    def test_dict_key_formats(
```

Comment:
I think with the the changes to test_override_get_value_element_method we no longer need this test. let me know if you think it's testing anything not covered by that test.

Reply:
So this test was still covering some scenarios that weren't covered elsewhere, however, after looking at it more closely, I think I finally found where it should live (I think)
See 2e545f9ec for the new test (hopefully now in the correct place)

Reply:
If you feel that is covering the functionality you want to test I am happy this being covered at that level.

Reply:
Yes, that sounds right to me => resolving